packages/run-protocol/src/vaultFactory/types.js

```diff
@@ -37,24 +37,21 @@
  */
 
 /**
- * @typedef  {Object} VaultFactory - the creator facet
+ * @typedef {Object} VaultFactory - the creator facet
  * @property {AddVaultType} addVaultType
- * @property {() => Promise<Array<Collateral>>} getCollaterals
+ * @property {() => Promise<Collateral[]>} getCollaterals
  * @property {() => Allocation} getRewardAllocation,
  * @property {() => Instance} getContractGovernor
  * @property {() => Promise<Invitation>} makeCollectFeesInvitation
  */
 
 /**
- * @callback ReallocateWithFee
- *
- * Transfer the indicated amount to the vaultFactory's reward
- * pool, taken from the `fromSeat`. Then reallocate over all the seat
- * arguments and the rewardPoolSeat.
- *
+ * @callback ReallocateWithFee Transfer the indicated amount to the
```

Comment:
This change makes it harder to distinguish the comment from the name when reading the source. Is there punctuation that can be inserted (' - ' for instance) that makes it more visible?

Reply:
Agree for code reviews. In an IDE it's still quite readable:
<img width="609" alt="Screen Shot 2022-03-24 at 9 26 08 AM" src="https://user-images.githubusercontent.com/21505/159963813-3d7d43ff-040c-4ae0-9a65-7421229f780f.png">
 
 I don't see an option to change this. I could file one but I'm not sure what to ask for… that an exception be made for callbacks?

I think this would be most in line with JSDoc and our other function comments:
```
/**
 * Transfer the indicated amount to the vaultFactory's reward pool, taken from
 * the `fromSeat`. Then reallocate over all the seat arguments and the rewardPoolSeat.
 *
 * @callback ReallocateWithFee
 * @param {Amount} amount
 * @param {ZCFSeat} fromSeat
 * @param {ZCFSeat} [otherSeat]
 * @returns {void}
 */
```
which renders on hover as,
<img width="785" alt="Screen Shot 2022-03-24 at 9 30 07 AM" src="https://user-images.githubusercontent.com/21505/159964658-5a775f19-9612-4296-9525-d81778f526c9.png">

Reply:
It's nice that VSCode does that, but WebStorm doesn't color code the types.js file.  

As a matter of fact, I now see that  at the call site for reallocateWithFee, WebStorm tells me that the type is `ReallocateWithFee`, and doesn't give any help on the parameters.  (This is at line 589 in vault.js.  WS shows no parameter help for `manager.reallocateWithFee(`, but the adjacent `transfer` shows a popup of `fromSeat, toSeat, fromLoses, toGains, key`.)  I've been noticing missing parameter help recently, but didn't notice that it was simultaneously present on other calls.

Reply:
> It's nice that VSCode does that, but WebStorm doesn't color code the types.js file.

That was just the style I was using. It did distinguish and allowed me to make them more distinct.

> at the call site for reallocateWithFee, WebStorm tells me that the type is ReallocateWithFee, and doesn't give any help on the parameters.

After some experimentation with @turadg, we found that `@callback` didn't pass the parameters through, but a direct declaration of the arguments did.  I will experiment so we don't have to describe all the parameters at each declaration.  (The particular example I was experimenting with was passed through several layers of API to get to where it was being used.)